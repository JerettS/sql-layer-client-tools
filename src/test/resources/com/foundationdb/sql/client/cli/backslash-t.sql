\t
SELECT 1,2,3,4;
\t
SELECT 1,2,3,4;
\t
SELECT 1 foo, 2 a;

CREATE TABLE t ( label int, long_label int, a int);
INSERT INTO t (label, long_label, a) values (1,2,3), (4,5,6), (7,8,9);
SELECT * FROM t;
\x
SELECT * FROM t;
\t on
\t ON
\t off
\t OFF
\t hi
