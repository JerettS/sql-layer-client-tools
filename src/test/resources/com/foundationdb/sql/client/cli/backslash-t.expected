Tuple only is on
          1 |          2 |          3 |          4
(1 row)

Tuple only is off
 _SQL_COL_1 | _SQL_COL_2 | _SQL_COL_3 | _SQL_COL_4
------------+------------+------------+------------
          1 |          2 |          3 |          4
(1 row)

Tuple only is on
   1 | 2
(1 row)

ROWS: 0

ROWS: 3

     1 |          2 | 3
     4 |          5 | 6
     7 |          8 | 9
(3 rows)

Expanded display is on
-
1
2
3
-
4
5
6
-
7
8
9

Tuples only is on
Tuples only is on
Tuples only is off
Tuples only is off
Wrong argument type: expected [on|off]